.github/workflows/npm-codescan-complete.yml

```diff
@@ -84,7 +89,17 @@ jobs:
           registry-url: https://npm.pkg.github.com/
           scope: '@htaic'
 
+      - name: Cache dependencies
+        id: cache
+        uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-node-${{ matrix.node-version }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-${{ matrix.node-version }}-${{ hashFiles('**/package-lock.json') }}
```

Comment:
I assume the team is using npm instead of yarn? Yarn actually automatically detects this kind of stuff and manages it for you (which is why we use all run commands

Reply:
Resolved By #25